scripts/verify-pallets-build.sh

```diff
@@ -66,6 +66,7 @@ rm -rf $BRIDGES_FOLDER/bin/rialto
 rm -rf $BRIDGES_FOLDER/bin/rialto-parachain
 rm -rf $BRIDGES_FOLDER/bin/.keep
 rm -rf $BRIDGES_FOLDER/deployments
+rm -rf $BRIDGES_FOLDER/docs
```

Comment:
Are you sure about this? I thought that maybe it'll help someone to understand what is this strange `bridges` folder about :) But it is up to you :)

Reply:
You're right. Maybe it's bnetter to kkep them. I chenged this to remove only `dockerhub-*`